src/docker-object-model.ts

```diff
@@ -0,0 +1,33 @@
+import { ToolRunner, IExecOptions, IExecSyncResult } from "./utilities/tool-runner";
+
+export class DockerExec{
+    private dockerPath: string;
+
+    constructor(dockerPath: string){
+        this.dockerPath = dockerPath;
+    };
+
+    public pullImage(args: string[], silent?: boolean) {
+        args = ['pull', ...args];
+        var result: IExecSyncResult = this.execute(args,silent);
+        if (result.stderr != '' && result.code != 0) {
+            throw new Error(`docker images pull failed with: ${result.error}`);
+        }
+    }
+
+    public inspectImage(args: string[], silent?: boolean): any {
+        args = ['inspect', ...args];
+        var result: IExecSyncResult = this.execute(args,silent);

```

Comment:
Prefer `let` over `var`.
Prefer `const` over `let` for read-only properties.

Reply:
Done